dali/pipeline/op_graph.h

```diff
@@ -30,7 +30,7 @@
 
 namespace dali {
 
-typedef int64 NodeID;
+using NodeID = int64_t;
```

Comment:
I thought you were renaming it in another PR, weren't you? In that case, you may leave the typedef here and spare yourself one conflict to resolve while rebasing...

Reply:
Everything is already rebased.